src/app/AlwaysOn.BackgroundProcessor/Services/EventHubProcessorService.cs

```diff
@@ -251,8 +251,7 @@ private Task PartitionInitializingHandler(PartitionInitializingEventArgs eventAr
             try
             {
                 ownedPartitions.TryAdd(eventArgs.PartitionId, DateTime.UtcNow);
-                _logger.LogInformation("Initialized partition: {partitionId}", eventArgs.PartitionId);
-                _logger.LogInformation("Now owning {numberOfOwnedPartitions} partition(s)", ownedPartitions.Count);
+                _logger.LogInformation("Initialized partition: {partitionId} Now owning {numberOfOwnedPartitions} partition(s)", eventArgs.PartitionId, ownedPartitions.Count);
```

Comment:
👍

Reply:
I'm not sure whether I can resolve this conversation :D